### PR TITLE
Windows Native Install Docs

### PR DESCRIPTION
**Pull request by dchw**

### CODE REVIEW

File: docs/alt-installation.md [hunk 1]
```diff
@@ -129,7 +129,28 @@ CentOS users can use our rpm repo to install Earthly.
...
 
+{% hint style='danger' %}
+##### Important
+
+Our native Windows release is currently in the **Experimental** stage.
+
+* The release ships with known issues. Many things work, but some don't.
+* Check the [GitHub tracking issue](https://github.com/earthly/earthly/issues/1031) for any known problems.
+* Give us feedback on [Slack](https://earthly.dev/slack).
+  {% endhint %}
```
Comment (vladaionescu) [inline docs/alt-installation.md:142]:
```suggestion

{% endhint %}
```

File: docs/alt-installation.md [hunk 2]
```diff
@@ -129,7 +129,28 @@ CentOS users can use our rpm repo to install Earthly.
...
+
+To install the Windows release, simply download the binary here; and ensure it is within your `PATH`.
+
+To add `earthly` to your `PATH` environment variable:
```
Comment (vladaionescu) [inline docs/alt-installation.md:146]:
```suggestion
To add `earthly.exe` to your `PATH` environment variable:
```

File: docs/alt-installation.md [hunk 3]
```diff
@@ -129,7 +129,28 @@ CentOS users can use our rpm repo to install Earthly.
...
+
+1. Search and select: System (Control Panel)
+2. Click the Advanced system settings link.
+3. Click Environment Variables. In the "System Variables" section, select the PATH environment variable and click Edit.
+   * If the PATH environment variable does not exist, click New.
+4. In the Edit window, specify the value of the PATH environment variable, and Click OK.
+5. Close and reopen any existing terminal windows, so they will pick up the new `PATH`.
```
Comment (vladaionescu) [inline docs/alt-installation.md:153]: Would it be difficult to host a Chocolatey package and/or a windows installer?

Maybe this is a later thing though - getting an `.exe` out there is definitely the more critical piece.

**Review by vladaionescu — APPROVED** [review]

File: docs/alt-installation.md (hunk 3 above)
Comment (dchw) [reply docs/alt-installation.md:154]: Later thing for sure. Chocolatey + the new WinGet would be probably important.

Packaging up an `.msi` installer should be fairly easy. I've never distributed Windows packages via Chocolatey or WinGet before; that would probably happen after I make an MSI.

**Pull request merged**